src/02-table-tests/index.test.ts

```diff
@@ -1,17 +1,27 @@
 // Uncomment the code below and write your tests
-/* import {  simpleCalculator, Action } from './index';
+import { simpleCalculator, Action } from "./index";
 
 const testCases = [
-    { a: 1, b: 2, action: Action.Add, expected: 3 },
-    { a: 2, b: 2, action: Action.Add, expected: 4 },
-    { a: 3, b: 2, action: Action.Add, expected: 5 },
-    // continue cases for other actions    
-]; */
+  { a: 5, b: 7, action: Action.Add, expected: 12 },
+  { a: 5, b: 7, action: Action.Subtract, expected: -2 },
+  { a: 5, b: 7, action: Action.Multiply, expected: 35 },
+  { a: 6, b: 3, action: Action.Exponentiate, expected: 216 },
+  { a: 6, b: 3, action: Action.Divide, expected: 2 },
+  { a: 6, b: null, action: Action.Exponentiate, expected: null },
+  { a: 6, b: null, action: Action, expected: null }
+];
 
-describe('simpleCalculator', () => {
-  // This test case is just to run this test suite, remove it when you write your own tests
-  test('should blah-blah', () => {
-    expect(true).toBe(true);
-  });
-  // Consider to use Jest table tests API to test all cases above
+describe("simpleCalculator", () => {
+  test.each(testCases)(
+    "a: $a, b: $b, action: $action should return $expected",
+    ({ a, b, action, expected }) => {
+      const result = simpleCalculator({ a, b, action });
+
+      if (Number.isInteger(expected)) {
```

Comment:
Why do you expect only integer?
```suggestion
      if (typeof expected === 'number') {
```